server/actions_versions.go

```diff
@@ -252,7 +252,7 @@ func (s *RegistryServer) UpdateApiVersion(ctx context.Context, req *rpc.UpdateAp
 	}
 
 	if err := saveVersion(ctx, client, version); err != nil {
-		return nil, internalError(err)
+		return nil, err
```

Comment:
Why is this not wrapping the err with internalError()? ... is this being caught at a higher level and handled there?

Reply:
The helper method already wraps errors as needed. In this case the only error code returned is internal so this would just be a double-wrap. Some other helpers (notably ones that read data) need to return several error codes depending on the error, so I thought it would make more sense for the helpers to be responsible for setting error codes.

Reply:
SGTM, thanks!